src/app/progress/Progress.tsx

```diff
@@ -452,6 +452,9 @@ class Progress extends React.Component<Props, State> {
 
     filteredItems.sort(
       chainComparator(
+        compareBy((item) => {
+          return item.expirationDate ? new Date(item.expirationDate) : new Date(8640000000000000);
```

Comment:
When the counter reaches zero the message in-game is something like "this bounty will disappear when you go to another area".
So it's possible to have a valid bounty with zero seconds left.
In this case the check above would return false and assign the "infinite" date, thus sorting this bounty down the list, right? Or am I missing something?

Reply:
I think what’ll actually happen is the expiration will be in the past, so it’ll be at the very top of the list. We should definitely change the text though.